crates/eframe/src/web/mod.rs

```diff
@@ -174,6 +174,13 @@ fn set_cursor_icon(cursor: egui::CursorIcon) -> Option<()> {
 /// Set the clipboard text.
 fn set_clipboard_text(s: &str) {
     if let Some(window) = web_sys::window() {
+        if !window.is_secure_context() {
+            log::error!(
```

Comment:
```suggestion
            log::debug!(
```

Reply:
I think it's fine to log at least a warning here. Something is clearly wrong when this occurs and the developer should have a chance to see this error message so they know what's wrong.